tiledb/api/c_api/datatype/datatype_api_enum.h

```diff
@@ -114,4 +114,8 @@
   TILEDB_DATATYPE_ENUM(BLOB) = 40,
   /** Boolean */
   TILEDB_DATATYPE_ENUM(BOOL) = 41,
+  /** std::byte */
+  TILEDB_DATATYPE_ENUM(GEOM_WKB) = 42,
+  /** std::byte */
+  TILEDB_DATATYPE_ENUM(GEOM_WKT) = 43,
```

Comment:
```suggestion
  /**
   * Geometry data in well-known binary (WKB) format.
   * Corresponds to std::byte.
   */
  TILEDB_DATATYPE_ENUM(GEOM_WKB) = 42,
  /**
   * Geometry data in well-known text (WKT) format.
   * Corresponds to std::byte.
   */
  TILEDB_DATATYPE_ENUM(GEOM_WKT) = 43,
```

What is the text encoding of the `GEOM_WKT` type? UTF-8?